skfem/mesh/mesh.py

```diff
@@ -22,8 +22,7 @@ class Mesh:
     _boundaries: Optional[Dict[str, ndarray]] = None
     _subdomains: Optional[Dict[str, ndarray]] = None
     elem: Type[Element] = Element
-    affine: bool = False
-    validate: bool = False  # unused; for backwards compatibility
```

Comment:
`validate` attribute is not doing anything. However, some tests tried to set it so I had it here during 3.0.0 release.